Sources/Regex/Legacy/VirtualMachine.swift

```diff
@@ -21,24 +21,29 @@ public struct MatchResult {
   }
 }
 
-/// VMs load RECode and run over Strings.
 public protocol VirtualMachine {
-  /// Declare this VM's motto and general life philosophy
+  associatedtype Program
```

Comment:
So this actually breaks the existential VirtualMachines in Swift 5.5. The only use I found was in exercises, where it's easy enough to just name it TortoiseVM